modules/nested-elements/assets/js/editor/nested-repeater/views/widget-repeater.js

```diff
@@ -0,0 +1,90 @@
+const WidgetView = elementor.modules.elements.views.Widget;
+
+/**
+ * @extends {BaseElementView}
+ */
+export class WidgetRepeater extends elementor.modules.elements.views.BaseElement {
+	initialize() {
+		this.config = elementor.widgetsCache[ this.options.model.attributes.widgetType ];
+
+		this.once( 'container:created', this.onContainerCreatedOnce.bind( this ) );
+
+		WidgetView.prototype.initialize.apply( this, arguments );
+	}
+
+	events() {
+		const events = super.events();
+
+		events.click = ( e ) => {
+			const $parentsUntil = jQuery( e.target ).parentsUntil( '.elementor-widget-container' );
+
+			// Clicks higher then 3, are nested, should not be handled by current.
+			if ( 3 <= $parentsUntil.length ) {
```

Comment:
Don't guess, try to find a selector or a container property/constructor

Reply:
https://github.com/elementor/elementor/pull/16252/commits/5989c02dc5b21394867f70cb78e1679ca3740e92